scripts/us_census/pep/us_pep_sex/process.py

```diff
@@ -91,203 +104,325 @@ def _national_1900_1979(file_path: str) -> pd.DataFrame:
     Returns:
         df (pd.DataFrame) : cleaned dataframe.
     """
-
-    # extracting year from file path.
-    year = file_path[-8:-4]
-    # schema is changing after 1960
-    if int(year) < 1960:
-        # schema is changing after 1940
-        if int(year) < 1940:
-            df = pd.read_csv(file_path,
-                             thousands=',',
-                             engine='python',
-                             skiprows=7,
-                             skipfooter=92,
-                             header=None)
+    try:
+        # extracting year from file path.
+        year = file_path[-8:-4]
+        # schema is changing after 1960
+        if int(year) < 1960:
+            # schema is changing after 1940
+            if int(year) < 1940:
+                df = pd.read_csv(file_path,
+                                 thousands=',',
+                                 engine='python',
+                                 skiprows=7,
+                                 skipfooter=92,
+                                 header=None)
+            else:
+                df = pd.read_csv(file_path,
+                                 thousands=',',
+                                 engine='python',
+                                 skiprows=7,
+                                 skipfooter=102,
+                                 header=None)
+            df.columns = [
```

Comment:
pls use remap instead of assuming column order.

Reply:
As discussed over chat rename method doesn't gives an upper hand since that approach has also demands to assume the rows and cols position.